patches/chrome-browser-ui-views-tabs-tab_style_views.cc.patch

```diff
@@ -1,16 +1,48 @@
 diff --git a/chrome/browser/ui/views/tabs/tab_style_views.cc b/chrome/browser/ui/views/tabs/tab_style_views.cc
-index 950c967b5f87a6e3f531e3155a420002a7510d3a..a9b5b74a1a5c345727c1deb631fce7fcbb2762dc 100644
+index 950c967b5f87a6e3f531e3155a420002a7510d3a..ad84681855bc39f55796c8740f490b998e2bc68a 100644
```

Comment:
Can't avoid patching. `GM2TabStyle` is defined in anonymous namespace.

Reply:
@simonhong: wasn't sure this was going to work, so had to try it out and it seems to be working. What do you think of subclassing instead like this: https://github.com/brave/brave-core/commit/4c19f3e0674dc0e460e5a0885fc659fe8d3a4aac

Reply:
Thanks @mkarolin ! Overriding `CreateForTab` is great :)